### ISSUE TITLE
Reconciler "Move" issue

### ISSUE THREAD

**Issue by sheva-serga** [jtheisen/reconciler#6]
I found 1 small issue in your library while implementing a new feature. 

When you trying to move some items from the collection of one parent (EF navigation property) to the collection of another parent in the scope of the same DB Context, the Reconciler just removes moved items from the DB.

I already created a fix of this issue and unit test for it, but I can't push my branch to this Repo.  
Please give me permissions, then I'll be able to push changes and create pull-request (which you will be able to review). 

Thanks in advance.

**Comment by jtheisen** [comment]
I don't think you need permission to do a pull request. Tell me if I'm wrong.

**Comment by sheva-serga** [comment]
Sorry, but currently I can't publish my local branch to this repository. 
```
git.exe push --progress "origin" bugfix/fix-reconcile-for-deleted-entry

remote: Permission to jtheisen/reconciler.git denied to sheva-serga.
fatal: unable to access 'https://github.com/jtheisen/reconciler.git/': The requested URL returned error: 403
```

**Comment by jtheisen** [comment]
The GitHub-way of doing it is to fork the repo (click on the fork icon on the top right), and set that as your new origin on your checkout. Then push there and create a pull-request, which I can then merge in or comment on.

**Comment by sheva-serga** [comment]
@jtheisen , thanks a lot for your reply. Please review my PR.

**Comment by jtheisen** [comment]
I released a new NuGet package, 0.2.3.

I don't know what that was with the version issue that made the tests fail. The version it's complaining about is normally changed by the `appveyor.yml`'s version anyway, so I'm not sure why the default version was an issue.
